Makefile

```diff
@@ -25,7 +25,12 @@ vet:
 	go vet ./...
 
 lint:
+ifndef $(shell command -v $(GOLANGCILINT) -v dot 2> /dev/null)
```

Comment:
Warn user about missing golangci-lint installation, but don't exit.

Reply:
Why not exit? The linting step seems useful, especially for outside contributors.

Reply:
My hope was that this warning would provide an easier way to call to action actually installing golangci-lint on the users machine nstead of:
```
make: golangci-lint: No such file or directory
```
But I see where you're getting at. Would you prefer that we log the installation instructions and then exit?